streams-contrib/streams-provider-instagram/src/main/java/org/apache/streams/instagram/serializer/util/InstagramActivityUtil.java

```diff
@@ -21,18 +21,11 @@
 
 import org.apache.streams.exceptions.ActivityConversionException;
 import org.apache.streams.exceptions.ActivitySerializerException;
-import org.apache.streams.instagram.pojo.Comment;
-import org.apache.streams.instagram.pojo.Comments;
-import org.apache.streams.instagram.pojo.Images;
-import org.apache.streams.instagram.pojo.Media;
-import org.apache.streams.instagram.pojo.UserInfo;
-import org.apache.streams.instagram.pojo.UserInfoCounts;
-import org.apache.streams.instagram.pojo.Videos;
+import org.apache.streams.instagram.pojo.*;
 import org.apache.streams.pojo.extensions.ExtensionUtil;
 import org.apache.streams.pojo.json.Activity;
 import org.apache.streams.pojo.json.ActivityObject;
 import org.apache.streams.pojo.json.Image;
-import org.apache.streams.pojo.json.ImageParent;
```

Comment:
How does this change relate to the version upgrade of `jsonschema2pojo` dependency?

I can't find this class anywhere in the `streams` repo, but also don't see evidence of it being removed in this PR. Was this dependency coming from the `org.apache.streams.plugins_streams-plugin-pojo` Maven dependency that's also being removed?

Reply:
yes - the prior generated sources was creating this derived class, but improving the schemata for/near MediaItem got rid of it.